### DISCUSSION TITLE
Welcome to lmt Discussions!

### DISCUSSION THREAD

**Discussion by driusan** [driusan/lmt#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AERodgers** [comment]
### Windows 10 Issue?

Hi,

I've been trying to install this on Window 10. I followed the instructions, but therewas an issue with ```go build```, so I tried the following: 

```
go build main.go
ren "main.exe" "lmt.exe"
```

This  seemed to work.

I then put a sample .md file into the lmt (test.praat.md) folder and ran ```lmt test.praat.md```.

Unfortunately, this did nothing.

My colleagues in the lab were able to get it to work in a Linux environment on [colab](https://colab.research.google.com/drive/1Q1736V_LCcnmS4-6O-WiDrLdf9K55cnq?usp=sharing)

***So, TL:DR, Wondering if ```lmt``` is designed to work on Linux only, if there are any known issues for Windows 10, or if I'm missing a step somewhere in the process.***

I hope I can get it working. It looks really useful.

**Reply by driusan** [reply]
I think I've used it on Linux, MacOS, Plan 9, and DragonFlyBSD so it's certainly not supposed to only work on Linux, but I don't have access to a Windows machine to test. I can't think of anything in the code that should prevent it from working on Windows.

If it's possible to put your "test.praat.md" file somewhere public (like a gist) I can check if it works here.

**Comment by AERodgers** [comment]
Thanks for getting back so fast. Here's a [link](https://github.com/AERodgers/Praat-Scripts/edit/master/test.praat.md) to the test file.

**Reply by driusan** [reply]
It seems like there's a bug in lmt with Windows line endings. I created issue #24 but don't have time to dig into it right now. As a workaround, if it's possible to save your file with unix style line endings I think that should solve your problem.

**Comment by AERodgers** [comment]
Hi. Thanks. I'll try that out.
